src/components/interaction-designer/BuilderCanvas.vue

```diff
@@ -23,6 +26,50 @@
 
     computed: {
       ...mapState('flow', ['flows']),
+
+      canvasHeight() {
+        const defaultHeight = 1500
```

Comment:
Would it be better to use `screen.availHeight` instead of hardcoding the height to a number?

Reply:
This is a great proposition @poojahpatil90 , I'm making a change